cardano-node-chairman/src/Testnet/Shelley.hs

```diff
@@ -75,7 +75,7 @@ rewriteGenesisSpec supply =
   rewriteObject
     $ HM.insert "activeSlotsCoeff" (toJSON @Double 0.1)
     . HM.insert "securityParam" (toJSON @Int 10)
-    . HM.insert "epochLength" (toJSON @Int 1500)
+    . HM.insert "epochLength" (toJSON @Int 1000)
```

Comment:
This is fine but we do also need to specify the slot length for Shelley. The unit for this is seconds. So you want e.g. 0.2 seconds.

Reply:
Added `slotLength`.